python/dolma/cli/mixer.py

```diff
@@ -40,7 +40,7 @@ class SpanReplacementConfig:
         default=None,
         help="Maximum score for the span to be replaced. Either min_score or max_score must be specified.",
     )
-    replacement: str = field(default="", help="Replacement for the span")
```

Comment:
Can we have minimal documentation for new sintax with `${}`?...

Reply:
Will do.